FrontEnd/mvnw.cmd

```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```

Comment:
What language is this? Can we remove this file?

Reply:
Used this file  to deploy to App Engine but not using it any more. deleted